From 767923525297c83676e6480b09e7f479e5d23539 Mon Sep 17 00:00:00 2001
From: James Davila <jamesddavila@gmail.com>
Date: Thu, 6 Sep 2018 13:32:50 -0400
Subject: [PATCH] fix-large-file-check

---
 configure.ac | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/configure.ac b/configure.ac
index e41baa3..a416dc6 100644
--- a/configure.ac
+++ b/configure.ac
@@ -183,16 +183,6 @@ LFS_CFLAGS=''
 if test "$enable_largefile" != no; then
     if test "$ac_cv_sys_file_offset_bits" != 'no'; then
 	LFS_CFLAGS="$LFS_CFLAGS -D_FILE_OFFSET_BITS=$ac_cv_sys_file_offset_bits"
-    else
-	AC_MSG_CHECKING(for native large file support)
-	AC_RUN_IFELSE([AC_LANG_SOURCE([#include <unistd.h>
-	  int main (int argc, char **argv)
-	  {
-	      exit(!(sizeof(off_t) == 8));
-	  }])],
-	[ac_cv_sys_file_offset_bits=64; AC_DEFINE(_FILE_OFFSET_BITS,64)
-	 AC_MSG_RESULT(yes)],
-	[AC_MSG_RESULT(no)])
     fi
     if test "$ac_cv_sys_large_files" != 'no'; then
 	LFS_CFLAGS="$LFS_CFLAGS -D_LARGE_FILES=1"
